packages/atlas/src/components/_navigation/SidenavViewer/SidenavViewer.tsx

```diff
@@ -22,6 +23,25 @@ export const SidenavViewer: React.FC = () => {
     updateChannelFollowing(id, false)
   }
 
+  const { signIn, activeMemberId, activeAccountId, extensionConnected } = useUser()
+
+  const isLoggedIn = !!activeAccountId && !!activeMemberId && !!extensionConnected
+
+  const buttonsContent = !isLoggedIn ? (
+    <Button size="large" icon={<SvgActionMember />} onClick={signIn}>
```

Comment:
```suggestion
    <Button icon={<SvgActionMember />} onClick={signIn}>
```
![Zrzut ekranu 2022-02-3 o 18 44 09](https://user-images.githubusercontent.com/13706246/152398728-a934fbf4-8093-453b-9fad-445aabee63ef.png)

Reply:
Corrected :)